app/posts/claim-general-mentor-training-funding/2022-06-26-general-mentor-training-grant-guidance-and-claim-form.md

```diff
@@ -0,0 +1,123 @@
+---
+title: General mentor training grant guidance and claim form
+description: Guidance page summarising ITT gerneral mentor funding and claim form.
+date: 2022-06-26
+---
+
+
+[The initial teacher training (ITT) market review report](https://www.gov.uk/government/publications/initial-teacher-training-itt-market-review-report) was published in July&nbsp;2021. After public consultation, the DfE published their [response to the ITT market review](https://www.gov.uk/government/consultations/initial-teacher-training-itt-market-review) in December&nbsp;2021.
+
+The recommendations which need to be followed from the 2024/25 academic year onwards, include requirements for:
+
+- mentors to train take part in 20 hours of initial training and 6 hours of top-up training in subsequent years
+- ITT providers to have lead mentors or a lead mentorship team
+- trainees to take part in ’Intensive training and practice’ sessions (4 weeks for postgraduate courses and 6 weeks for undergraduate courses)
+
+## ITT reform funding
+
+To support providers and schools in meeting the new requirements the DfE is making grant funding available.
+
+For schools the funding is to help cover the cost of classroom cover while mentors train. In the 2024/25 academic year, general mentors will need to complete 20&nbsp;hours of initial mentor training. In subsequent years general mentors who have completed the 20&nbsp;hours will need 6&nbsp;hours of training.
+
+[See full details of the ITT reform funding.](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1077177/Initial_Teacher_Training__ITT__Reform_Funding_Guidance_160522.pdf)
```

Comment:
```suggestion
[See full details of the ITT reform funding (PDF).](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1077177/Initial_Teacher_Training__ITT__Reform_Funding_Guidance_160522.pdf)
```

Do you think this should say it's going to an external site or the file size? should it force a new window?

Reply:
Yeah, forced new tab.